src/PiTop.Algorithms/VelocitySensor.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Reactive;
+using System.Reactive.Concurrency;
+using System.Reactive.Linq;
+using UnitsNet;
+
+namespace PiTop.Algorithms
+{
+    public class VelocitySensor : IDisposable
+    {
+        private readonly IDisposable _sampler;
+        private (DateTimeOffset Timestamp, Acceleration Accelecation)? _previousSample;
+        private IScheduler _scheduler;
+
+        public VelocitySensor(TimeSpan samplingFrequency, Func<Acceleration> accelerationSampler, IScheduler samplingScheduler = null)
+        {
+            Velocity = Speed.Zero;
```

Comment:
Please document that speed is assumed zero at sensor creation. Might be good to be able to point to that.